content/docs/api-reference/make_checkpoint.md

```diff
@@ -52,42 +52,28 @@ Let's consider the following `dvc.yaml` file:
 
 ```yaml
 stages:
-  every100:
-    cmd: python iterate.py
+  train:
+    cmd: python train.py
     outs:
-      - int.txt:
+      - model:
           checkpoint: true
 ```
 
-The code in `iterate.py` will execute continuously increment an integer number
-saved in `int.txt` (starting at 0). At 0 and every 100 loops, it makes a
-checkpoint for [DVC experiments]:
+The code in `train.py` will train a model upto certain epochs. At every 100 iterations,
+it saves the model to `model`, evaluates it and makes a checkpoint for [DVC experiments]:
 
 [dvc experiments]: /doc/user-guide/experiment-management#experiments
 
 ```py
-import os
-
 from dvc.api import make_checkpoint
 
-while True:
-    try:
-        if os.path.exists("int.txt"):
-            with open("int.txt", "r") as f:
-                i_ = int(f.read()) + 1
-        else:
-            i_ = 0
-
-        # ... do something meaningful
-
-        with open("int.txt", "w") as f:
-            f.write(f"{i_}")
+for epoch in range(epochs):
+    train(model, x_train, y_train)
```

Comment:
I think `model`/`x_train`/`y_train`/`x_test`/`y_test` are common namings that we don't have to show them in the code.